src/backend/utils/adt/agtype.c

```diff
@@ -2654,7 +2654,8 @@ Datum agtype_to_int4(PG_FUNCTION_ARGS)
         (agtv.type != AGTV_FLOAT &&
          agtv.type != AGTV_INTEGER &&
          agtv.type != AGTV_NUMERIC &&
-         agtv.type != AGTV_STRING))
+         agtv.type != AGTV_STRING &&
+         agtv.type != AGTV_BOOL))
```

Comment:
All ifs need to have {} even for 1 line of code. If you are modifying old code that doesn't follow that standard, please update it. No need to update everything, just the contents of the function.

Reply:
Since none of the 'if', 'else if' conditions have {} for 1 line of code inside these functions, do I add to all the conditions that I have not modified or to only the ones I have changed?

Reply:
If they are in the function that you are working on, that would be great. We're slowly trying to update these as we come across them. I wouldn't do more than just the function(s) that you are working on.